proto/platform/registry/v1/registry.platform.proto

```diff
@@ -404,6 +404,9 @@ message PackageReference {
 message VulnerabilitiesDiff {
   repeated VulnerabilityReference added = 1;
   repeated VulnerabilityReference removed = 2;
+  chainguard.platform.tenant.Scanner scanner = 3;
+  // vulnerability_db_last_build_time holds the time when the vulnerability database was built last time.
+  google.protobuf.Timestamp vulnerability_db_last_build_time = 4;
```

Comment:
I was thinking more of having: 

```
message VulnScannerDiff {
  ChangedVulnScanner changed = 1;
}

message ChangedVulnScanner {
  message ChangedVulnScannerName {
    string current = 1;
    string previous = 2;
  }
  ChangedVulnScannerName changedName = 1;

  message ChangedVulnScannerVersion {
    string current = 1;
    string previous = 2;
  }
  ChangedVulnScannerVersion changedVersion = 2;

  message ChangedVulnScannerVulnerabilityDbLastBuildTime {
    google.protobuf.Timestamp current = 1;
    google.protobuf.Timestamp previous = 2;
  }
  ChangedVulnScannerVulnerabilityDbLastBuildTime changedVulnerabilityDbLastBuildTime = 2;
}

```
where `VulnScannerDiff` is in `DiffImageResponse`.